ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -74,6 +77,16 @@ export function D3Blueprint(container) {
                     opacity: 0
                 }
             },
+        warning: {
+            image: {
+                class: 'node-warning',
+                width: 24,
+                height: 24,
+                x: -65,
+                y: 15,
+                'xlink:href': warningIcon
```

Comment:
I don't know that syntax, what does it do?

Reply:
This is used to load an image. It was inspire from another icon (you can find it in the same file)